diff --git a/node_modules/playwright-core/lib/coreBundle.js b/node_modules/playwright-core/lib/coreBundle.js
index d9041dd..5b3e814 100644
--- a/node_modules/playwright-core/lib/coreBundle.js
+++ b/node_modules/playwright-core/lib/coreBundle.js
@@ -43972,7 +43972,7 @@ var init_electron = __esm({
       }
       async launch(progress2, options) {
         let app = void 0;
-        let electronArguments = ["--inspect=0", "--remote-debugging-port=0", ...options.args || []];
+        let electronArguments = ["--inspect=0", ...options.args || []];
         if (import_os14.default.platform() === "linux") {
           if (!options.chromiumSandbox && electronArguments.indexOf("--no-sandbox") === -1)
             electronArguments.unshift("--no-sandbox");
